include/openmc/settings.h

```diff
@@ -86,6 +86,9 @@ extern "C" int64_t n_particles;              //!< number of particles per genera
 
 extern int64_t max_particles_in_flight; //!< Max num. event-based particles in flight
 
+extern "C" bool sort_fissionable_xs_lookups; //!< Sort fissionable material XS lookups in event-based mode?
+extern "C" bool sort_non_fissionable_xs_lookups; //!< Sort non-fissionable material XS lookups in event-based mode?
```

Comment:
Looks like this doesn't really need to be `extern "C"`, I think a plain `extern` will suffice

Reply:
Good point!